docs/source/install.rst

```diff
@@ -81,7 +91,7 @@ in the table below.
 +-------------------------------------------+--------------------------+
```

Comment:
Now that I look at it, I believe there was a reason I used this table format, since I used the other format in other places (see intro). I'm going to try to use the other formatting and see if it changes.

Reply:
Tried it out but the output did not change. If I have time tomorrow I'll look into seeing if I can change the column width.